### ISSUE TITLE
Text color with dark themes

### ISSUE THREAD

**Issue by beliys** [asbru-cm/asbru-cm#640]
**The color of text for connections on dark themes is poorly read.**

**To Reproduce**
1. Setup any dark theme for gtk app
2. Run asbru-cm
3. See text color in connections list

**Screenshots**
https://imgur.com/OkszR7x

**Environment (please complete the following information):**
 - OS: Manjaro-KDE latest
 - Ásbrú Version v6.2.1 from AUR (http://aur.archlinux.org/packages/asbru-cm-git)

**Comment by midniteslice** [comment]
I was just coming here with the same issue. 
Version 6.2.0 on kubuntu 20.04 has the same issue. Version 6.1.3 on fedora 31 running kde does not have the issue. 
I have tried many different themes with the same result.

**Comment by popxunga** [comment]
Hi ... 

I do use Ad-waita Dark theme and it looks good. 

My Asbru setup looks like this:

![image](https://user-images.githubusercontent.com/37672088/83652881-06cc4200-a5b3-11ea-805a-3487e301ec38.png)

![image](https://user-images.githubusercontent.com/37672088/83653025-33805980-a5b3-11ea-841d-a06bdab48436.png)

Let us know if that helps.

**Comment by beliys** [comment]
@popxunga , thanks, the "Look and Feel" setting helped me, but it would be better if the default settings were from the system theme.

**Comment by midniteslice** [comment]
That did it for me too. I agree with what beliys said about ti being default. 
Thank You!

**Comment by popxunga** [comment]
Indeed ... the default colors used may need some adjustments ...

**Comment by gfrenoy** [comment]
Not sure what you mean by the "default":

- when you first start Ásbrú (so with a brand new configuration directory)
- when you switch your GTK theme
- when you switch your Ásbrú Icons theme

There is no magic, we need to define the logic that needs to be implemented and I don't see here what to do exactly.

**Comment by popxunga** [comment]
Hi @gfrenoy,
From my point of view, the default text/icon colors should always be relative to the Asbru themes being used, only.
This may be more difficult as it looks on a 1st view, but as basic principle I would say that the logic behind this can be:
For the Asbru dark theme, light colors for text and icons should be used by default;
For the Default or Colored themes, dark colors for text and icons should be used by default.
If the user decides to use the System theme then is not an Asbru problem.

**Comment by beliys** [comment]
I agree with @popxunga , but when using a system theme, the colors should be taken from the theme that the system uses.

**Comment by popxunga** [comment]
Hi @beliys ... when the System theme is selected on Asbru, then Asbru should just pick up and use the colors defined in the theme.

**Comment by Mjolinir** [comment]
I agree with the gentlemen above, the colors, fonts and other theme related items should correspond to what the user has defined on the system by default. I like the idea of being able to override system defaults in the app, but the default should be what is set in the system. This allows for a consistent look and feel and makes the app feel more enterprise quality.

**Comment by gfrenoy** [comment]
Understood, there's definitely something to improve here ...

The main issue I see here is that we do set a default color.  Actually, we should have the option to "unset" the color.  Which would mean that this would use the system color.

Sorry could not find a lot of time to improve Ásbrú recently ; let me put this into the priority for the next release so I do forget.

In the meantime, any kind of contribution will be of course very welcome.

**Comment by AlexFolland** [comment]
Setting the icon theme to `asbru-dark` did not fix the connection name text color for me, and I don't see why it should, since it is called an icon theme.  Also, the "Save and Close" button in my interface no longer works after changing these settings, so I've been editing the yml file directly to change settings now, and nothing seems to have allowed the connection name text to become white.  I will note that all other text looks great though.  It's just that text in the connection list that's black on black.

**Comment by dennisTGC** [comment]
The setting does nothing on my system when asbru-cm is installed as a snap. I tried asbru-dark and system. I'm on asbru 6.4.0. Let me know if I can test any more. 

I'm on kubuntu 22.04.1

**Comment by dennisTGC** [comment]
The setting does nothing on my system. I tried asbru-dark and system. I'm on asbru 6.4.0. Let me know if I can test any more. 

I'm on kubuntu 22.04.1, asbrucm-loki 6.4.1

**Comment by gfrenoy** [comment]
This issue is related to a enhancement request that is not yet implemented.  It is somehow expected that it is not working yet...

**Comment by gfrenoy** [comment]
This has been improved on the loki branch.  Feel free to comment if not suitable.
The changes will be available in the next release.

**Comment by popxunga** [comment]
Looks good. Thank you !
